runelite-client/src/main/java/net/runelite/client/plugins/defaultworld/DefaultWorldPlugin.java

```diff
@@ -116,32 +115,32 @@ private void changeWorld(int newWorld)
 			return;
 		}
 
-		try
+		final WorldResult worldResult = worldService.getWorlds();
```

Comment:
This happens right at startup so are we sure the worlds will be actually loaded at this point? Maybe the world client could do blocking call when its being created.